lib/Graylog.js

```diff
@@ -424,4 +424,3 @@ Graylog.prototype.close = function (cb) {
 
 
 module.exports = Graylog;
-module.exports.graylog = Graylog; // deprecated
```

Comment:
This one line keeps API compatibility between the old and the new. That's why I added it to the re-implementation.

Reply:
Is it really necessary though? 
I did have the idea that this was supposed to be a major version bump, which means it won't be backwards compatible.